frontend/components/LanguageSelector.vue

```diff
@@ -35,7 +35,7 @@ const languageDisplay = (l: string) => {
 </script>
 
 <template>
-    <BccSelect v-model="model" :label="$t('language')">
+    <BccSelect  :required="true" v-model="model" :label="$t('language')">
```

Comment:
Here you can just do `required` instead of `:required="true"`.
Doesn't really matter in the end, but it is less verbose.